app/build.gradle

```diff
@@ -284,31 +284,31 @@ repositories {
 }
 
 dependencies {
-    implementation "com.android.support:appcompat-v7:$support_libraries_version"
```

Comment:
Could you update the constant here: https://github.com/mozilla-mobile/focus-android/blob/8097e857ddef1c51e3607a80889bcbf2fab13a82/build.gradle

instead of hard coding the new values?

Cheers!